rust/Cargo.lock

```diff
@@ -97,114 +97,6 @@ version = "0.7.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "8da52d66c7071e2e3fa2a1e5c6d088fec47b593032b254f5e980de8ea54454d6"
 
-[[package]]
```

Comment:
We will merge this into main once asaj/v2-deploy is merged. Comment to prevent auto-merge.